src/app/oktaAuthCallback/oktaAuthCallback.tpl.html

```diff
@@ -10,7 +10,8 @@ <h3 class="text-center">
                   <div>
                     <loading></loading>
                     <br/>
-                    <p translate>{{$ctrl.noticeToUser}}</p>
+                    <span ng-if="!$ctrl.isReadyToRedirect" translate>Authenticating...</span>
```

Comment:
Based on other code in the repo, you can put the text inside `<loading>`. Once #1196 merges, there will be an animation with the loading bars.

Reply:
The other loaders have the words appear before or above the loading icon. Which might be needed elsewhere, but for this, I think we need to put the loading icon above the text, so this is the natural flow many other sites take (Showing the loading icon before the text), as the loading icon catches the eye of the user. Then the text gives the user more context.